src/xo-app/new/import/index.js

```diff
@@ -0,0 +1,140 @@
+import ActionButton from 'action-button'
+import Component from 'base-component'
+import Dropzone from 'react-dropzone'
+import Icon from 'icon'
+import React from 'react'
+import _ from 'messages'
+import filter from 'lodash/filter'
+import map from 'lodash/map'
+import { Col, Row } from 'grid'
+import { importVm } from 'xo'
+
+import {
+  Card,
+  CardBlock,
+  CardHeader
+} from 'card'
+
+import {
+  SelectPool,
+  SelectSr
+} from 'select-objects'
+
+import {
+  formatSize,
+  noop
+} from 'utils'
+
+import styles from './index.css'
+
+export default class Import extends Component {
+  constructor (props) {
+    super(props)
+    this.state.files = []
+  }
+
+  _import = () => Promise.all(map(this.state.files, file =>
+    importVm({ sr: this.refs.selectSr.value.id, file }).catch(noop)
+  ))
+
+  _onDrop = files => {
+    this.setState({
+      files: filter(files, file => file.name.endsWith('.xva'))
+    })
+  }
+
+  _onCleanSelectedVms = () => {
+    this.setState({
+      files: []
+    })
+  }
+
+  _handleSelectedPool = pool => {
+    const srPredicate = pool !== ''
+      ? sr => sr.$pool === pool.id && sr.content_type === 'user'
+      : undefined
+
+    this.setState({
+      srPredicate
+    }, () => { this.refs.selectSr.value = pool.default_SR })
+  }
+
+  render () {
+    const { files, srPredicate } = this.state
+
+    return (
+      <Card>
+        <CardHeader>
+          <Icon icon='import' /> {_('newImport')}
+        </CardHeader>
+        <CardBlock>
+          <form id='import-form'>
+            <div className='form-group row'>
+              <label className='col-sm-2 form-control-label'>{_('vmImportToPool')}</label>
```

Comment:
I am not found of this col/label pattern, wouldn't it be possible to use a `label` inside a `Col`?